.devcontainer/devcontainer.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "Bun",
```

`name`にはプロジェクトにちなんだ名前を設定すると良いと思います。

https://containers.dev/implementors/json_reference/

```suggestion
  "name": "alg.tus-ricora.com",
```